src/test/java/com/wikia/webdriver/elements/mercury/old/MercuryAlertComponentObject.java

```diff
@@ -1,46 +1,50 @@
-package com.wikia.webdriver.elements.mercury.old;
-
-import com.wikia.webdriver.common.core.elemnt.Wait;
-
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
-import org.openqa.selenium.support.FindBy;
-import org.openqa.selenium.support.PageFactory;
-
-public class MercuryAlertComponentObject {
-
-  @FindBy(css = ".alert-notifications .alert-box")
-  protected WebElement alertBox;
-
-  private String alertMessage;
-
-  public enum AlertMessage {
-    NOT_EXISTING_REDIRECT(
-        "The link you followed is a redirect, but the page it directs to does not exist."),
-    NOT_EXISTING_CATEGORY("Category not found"),
-    NOT_EXISTING_SECTION("Section not found");
-
-    private String message;
-
-    AlertMessage(String message) {
-      this.message = message;
-    }
-
-    public String getMessage() {
-      return message;
-    }
-  }
-
-  private Wait wait;
-
-  public MercuryAlertComponentObject(WebDriver driver, AlertMessage message) {
-    this.wait = new Wait(driver);
-    this.alertMessage = message.getMessage();
-
-    PageFactory.initElements(driver, this);
-  }
-
-  public boolean isAlertMessageVisible() {
-    return wait.forTextInElement(alertBox, alertMessage);
-  }
-}
+package com.wikia.webdriver.elements.mercury.old;
+
+import com.wikia.webdriver.common.core.elemnt.Wait;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class MercuryAlertComponentObject {
+
+  @FindBy(css = ".alert-notifications .alert-box")
+  protected WebElement alertBox;
+
+  private String alertMessage;
+
+  public void setAlertMessage(AlertMessage message) {
+    this.alertMessage = message.getMessage();
+  }
+
+  public enum AlertMessage {
+    NOT_EXISTING_REDIRECT(
+        "The link you followed is a redirect, but the page it directs to does not exist."),
+    NOT_EXISTING_CATEGORY("Category not found"),
+    NOT_EXISTING_SECTION("Section not found");
+
+    private String message;
+
+    AlertMessage(String message) {
+      this.message = message;
+    }
+
+    public String getMessage() {
+      return message;
+    }
+  }
+
+  private Wait wait;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1213)